app/lib/pre_assembly/digital_object.rb

```diff
@@ -225,15 +219,13 @@ def generate_content_metadata
     end
 
     # Invoke the contentMetadata creation method used by the project
-    # The name of the method invoked must be "create_content_metadata_xml_#{content_md_creation--style}", as defined in the YAML configuration
-    # Custom methods are defined in the project_specific.rb file
     # if we are not using a standard known style of content metadata generation, pass the task off to a custom method
```

Comment:
maybe the last line of method comment for create_content_metadata can go away now - the code is clear and the comment is less than clear.

Reply:
whoops, while reviewing, we thought the removed lines were sufficient, but going over this again, i see what you're saying, @ndushay.  will remove in a small follow-on pr after merging this...

Reply:
#396